taxi_service/urls.py

```diff
@@ -22,4 +22,5 @@
 urlpatterns = [
     path("admin/", admin.site.urls),
     path("", include("taxi.urls", namespace="taxi")),
+    path("accounts/", include("django.contrib.auth.urls")),
 ] + static(settings.STATIC_URL, document_root=settings.STATIC_ROOT)
```

Comment:
Using `static()` to serve static files is suitable for development, but in a production environment, you should configure your web server (e.g., Nginx, Apache) to serve static files directly for better performance.